openstack/neutron/templates/seed.yaml

```diff
@@ -75,10 +75,10 @@ spec:
         role: cloud_network_admin
       - project: l3admin
         role: cloud_compute_admin
-{{- if and (.Values.interconnection.enabled) (.Values.interconnection.service_password) }}
+{{- if and (.Values.interconnection.enabled) }}
```

Comment:
Probably doesn't need the and (..) prefix anymore

Reply:
yes, exactly